makefile

```diff
@@ -175,6 +175,13 @@ build-ffis: dexrt-llvm
 	cp $(STACK_INSTALL_DIR)/lib/libDex.so python/dex/
 	cp $(STACK_INSTALL_DIR)/lib/libDex.so julia/deps/
 
+install-ffis: dexrt-llvm
+	$(STACK) build                     $(STACK_FLAGS) --work-dir .stack-work-ffis --force-dirty --flag dex:foreign --flag dex:optimized
+	$(STACK) install $(STACK_BIN_PATH) $(STACK_FLAGS) --work-dir .stack-work-ffis --force-dirty --flag dex:foreign --flag dex:optimized
+	$(eval STACK_INSTALL_DIR=$(shell $(STACK) path --work-dir .stack-work-ffis --local-install-root))
+	cp $(STACK_INSTALL_DIR)/lib/libDex.so python/dex/
+	cp $(STACK_INSTALL_DIR)/lib/libDex.so julia/deps/
```

Comment:
This is quite weird. The `stack install` invocation a few lines up will copy the Dex libraries into some directory specified by PREFIX, but those two copies will move the `libDex.so` into a subdirectory of the root. So it's not really installing any FFIs?

Reply:
My goal was just to have something that CI can invoke to get a directory from which `dex` on the command line and `import dex` in Python both work, which is a mix of what `make build-ffis` and `make install` do.  And to only compile Dex once in the process.  Renamed the target and added a comment, and also changed it to ignore PREFIX and install into `.`, to make it more clearly a "coherent" thing specialized to CI.